src/ds/test/serializer.cpp

```diff
@@ -117,7 +117,7 @@ void require_minimum_serialized_arity(Ts&&... ts)
   // Any serializer should produce at least as many sections as arguments (some
   // arguments may produce multiple sections)
   decltype(Serializer::serialize(ts...)) sections;
-  REQUIRE_NOTHROW(sections = Serializer::serialize(std::forward<Ts>(ts)...));
+  // REQUIRE_NOTHROW(sections = Serializer::serialize(std::forward<Ts>(ts)...));
```

Comment:
?

Reply:
Thanks for catching that, this was causing issues with libc++, looking into it again now.

Reply:
With the cmake lists fixed, we get a clearer error:

```
/usr/lib/llvm-7/bin/../include/c++/v1/tuple:616:5: note: copy assignment operator is implicitly deleted because 'tuple<const std::__1::shared_ptr<serializer::RawSection<int> >, const std::__1::shared_ptr<serializer::RawSection<int> > >' has a user-declared move constructor
    tuple(tuple&&) = default;
```